crates/holochain/src/core/workflow/app_validation_workflow/types.rs

```diff
@@ -0,0 +1,44 @@
+use std::convert::TryFrom;
+
+use holo_hash::AnyDhtHash;
+
+use crate::core::validation::OutcomeOrError;
+
+use super::AppValidationOutcome;
+
+#[derive(Debug)]
+/// The outcome of sys validation
+pub enum Outcome {
```

Comment:
This name feels too generic for a public type... Is AppValidationOutcome too verbose?

Reply:
I'm going to address this in a follow up PR that combines the sys and app validation outcomes into the OutcomeOrError type which will make this much clearer and not use errors